frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -42,14 +50,20 @@ const NameDescriptionField: React.FC<NameDescriptionFieldProps> = ({
   return (
     <Stack hasGutter>
       <StackItem>
-        <FormGroup label="Name" isRequired fieldId={nameFieldId}>
+        <FormGroup
+          label="Name"
+          isRequired
+          fieldId={nameFieldId}
+          helperText={isNameValid && nameHelperText}
```

Comment:
Why is the helper text conditionally displayed?
```suggestion
          helperText={nameHelperText}
```